core/src/main/java/projectzero/UMLClassYamlMapper.java

```diff
@@ -0,0 +1,22 @@
+package projectzero;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+public class UMLClassYamlMapper {
+    public List<UMLClass> read(String fileName) throws IOException {
+        ObjectMapper objectMapper = new ObjectMapper(new YAMLFactory());
+        return objectMapper.readValue(new File(fileName), new TypeReference<>() {
+        });
```

Comment:
Are these curley braces needed for this return statement?

Reply:
I can check.

Reply:
It is required.